aws/cf.go

```diff
@@ -497,23 +535,33 @@ func mapToManagedStack(stack *cloudformation.Stack) *Stack {
 		statusReason:      aws.StringValue(stack.StackStatusReason),
 		CWAlarmConfigHash: tags[cwAlarmConfigHashTag],
 		WAFWebACLID:       parameters[parameterLoadBalancerWAFWebACLIDParameter],
-	}
+		ExtraListeners:    extraListeners,
+		loadbalancerARN:   outputs.lbARN(),
+	}, nil
 }
 
 func findManagedStacks(svc cloudformationiface.CloudFormationAPI, clusterID, controllerID string) ([]*Stack, error) {
 	stacks := make([]*Stack, 0)
+	errors := make([]error, 0)
 	err := svc.DescribeStacksPages(&cloudformation.DescribeStacksInput{},
 		func(page *cloudformation.DescribeStacksOutput, lastPage bool) bool {
 			for _, s := range page.Stacks {
 				if isManagedStack(s.Tags, clusterID, controllerID) {
-					stacks = append(stacks, mapToManagedStack(s))
+					stack, err := mapToManagedStack(s)
+					if err != nil {
+						errors = append(errors, err)
+					}
+					stacks = append(stacks, stack)
```

Comment:
we would here either add empty stack (&Stack{}) or nil. We could think of a continue in the error path instead. 
I am not sure if we need empty stacks or nil stacks for later use, maybe for deletion or new creation.